src/internal/methods/NeoEspBitBangMethod.cpp

```diff
@@ -28,7 +28,7 @@ License along with NeoPixel.  If not, see
 
 #if defined(ARDUINO_ARCH_ESP8266) || defined(ARDUINO_ARCH_ESP32) 
 
-#include "..\NeoUtil.h"
```

Comment:
thanks for catching this.  Some platforms this works fine as is, others require this forward slash which works on all platforms.